### DISCUSSION TITLE
how to stop dag initial and historical run

### DISCUSSION THREAD

**Discussion by NandhakumarSL** [apache/airflow#34540] [Q&A]
i created an 15 dags if my system goes shutdown and then after reboot it triggers all dags that time my cpu goes high i can't control it will overload my system it goes to crash again it will happend how to stop intial run and historical run . please help me 
i'm using airflow 2.5.3
backend database server is mysql

**Comment by Taragolis** [comment]
Try to run only Airflow webserver and do not run Airflow schedulers/workers/dag_processors. 
You could mark all DAG runs as failed in the UI, in additional you could pause DAGs
